ModelLibrary/Process/AttachMailToBP.cs

```diff
@@ -300,7 +300,18 @@ private void GetMails(UserInformation user, int AD_User_ID, int AD_Client_ID, in
                         //  {
                         try
                         {
-                            string sql = "SELECT " + tableName + "_ID " + " , C_BPartner_ID,Name,value " + "FROM " + tableName + " WHERE lower(Email) like " + "'%" + from.Trim().ToLower() + "%'";
+                            string sql;
+                            //changes done by Emp id:187
+                            //If sender is lead it checks this query
+                            if (sender == "C_Lead")
+                            {
+                                sql = "SELECT C_Lead_ID, C_Bpartner_ID,Name,DocumentNo as value FROM C_Lead WHERE lower(Email) like " + "'%" + from.Trim().ToLower() + "%'";
```

Comment:
SQL Keywords should be in capital letters 
e.g. LIKE, AS, LOWER, SELECT, FROM, WHERE etc.